content/Video_Display.md

```diff
@@ -670,10 +670,11 @@ sprite in OAM (\$FE00-\$FE03) has the highest priority, and so on. In
 Non-CGB mode, the smaller the X coordinate, the higher the priority. The
 tie breaker (same X coordinates) is the same priority as in CGB mode.
 
-The priority calculation between sprites disregards OBJ-to-BG Priority
-(attribute bit 7). Only the highest-priority nonzero sprite pixel at any
+After sprites priorities have been determined, in those areas where they overlap
+the OBJ-to-BG Priority (attribute bit 7) of the lower priority sprites is ignored.
```

Comment:
I think this is confusing, and would put it after the following sentence: the OBJ-to-BG priority doesn't get ignored if there's overlap at all, it just only gets considered *afterwards*—as the next sentence explains. I believe that swapping the order would remove any ambiguity.

Reply:
How is it considered afterwards? Sorry I don't understand.

Reply:
As you may know if you've read the docs about the FIFO, the GB actually has a FIFO for BG pixels and another one for OBJ pixels.

OBJ-to-OBJ priority is considered when inserting pixels into the OBJ FIFO (to decide whether an incoming pixel should overwrite the current one or not).

OBJ-to-BG priority is considered each time the FIFOs are shifted, to select of which FIFO to keep the pixel.

Therefore, OBJ-to-OBJ priority is considered at an earlier stage than the OBJ-to-BG priority, and the OBJ-to-OBJ priority process does not consider the OBJ-to-BG bit. The combination of these is what causes the quirk.